core/survey/survey.js

```diff
@@ -47,6 +47,7 @@ export const newSurvey = ({
     [SurveyInfo.keys.languages]: languages,
     [SurveyInfo.keys.labels]: label ? { [languages[0]]: label } : {},
     [SurveyInfo.keys.srs]: srs && srs.length > 0 ? srs : [R.omit([Srs.keys.wkt], Srs.latLonSrs)],
+    [SurveyInfo.keys.samplingPolygon]: SurveyDefaults.getSamplingPolygonDefaults(),
```

Comment:
I wouldn't include the sampling polygon object to all the surveys; it will be used only in some cases and we will end up with a lot of surveys having this "dirty" information...